### DISCUSSION TITLE
Unable to Access Firefox due to Older Version of Firefox Error

### DISCUSSION THREAD

**Discussion by m-farhana** [llm-workflow-engine/llm-workflow-engine#219] [Q&A] [answered]
Hello all, 

I have been working with this ChatGPT Wrapper for a few months now. While it was working fine all this while, just yesterday, I ran into an error where I was unable to open Firefox due to the error message shown below. 
<img width="748" alt="Screenshot 2023-03-08 at 4 38 59 PM" src="https://user-images.githubusercontent.com/86278922/223669922-3acfa345-1f8d-4fcc-ad69-f4c085d6c43d.png">

Even though I pressed "Create New Profile", an error message showed up in my terminal as shown in the first attachment and I was unable to open ChatGPT in the Firefox window. 
![photo_2023-03-08 16 36 29 (1)](https://user-images.githubusercontent.com/86278922/223669999-0622e0df-a233-4cf0-bf0c-97a58cb80588.jpeg)

Even though I had this error, for a short while, I was still able to access and use ChatGPT. However, soon after, I began to get the following error message: 
Your ChatGPT session is not usable. 
* Run this program with the `install` parameter and log in to ChatGPT. 
* If you think you are already logged in, try running the `session` command.

I believe that I am supposed to login to my ChatGPT session on Firefox. However, as I am unable to open Firefox using 'chatgpt install', I am unsure as to how to proceed further and I seek your guidance with the next steps.

Thank you and I look forward to getting some feedback!

**Comment by mmabrouk** [comment, marked as answer]
I would remove and reinstall playwright and the firefox in playwright. I would maybe first remove all tmp folders used by playwright first (i.e. last line in the screenshot).
Hope that helps

**Reply by m-farhana** [reply]
Hi, thank you for your reply. This was helpful!
